styleguide/source/_patterns/01-molecules/01-layouts/07-layout-five-up-left/_layout-five-up-left.scss

```diff
@@ -0,0 +1,57 @@
+// Placeholder blocks in layout molecules.
+.layout_five-up-left {
+  &.layout-example .layout_container {
+    background-color: $black-7;
+  }
+  .layout_container {
+    display: block;
+    margin-bottom: 1em;
+    padding: 2em;
+    text-align: center;
+  }
+}
+// Layout default setup.
+
+.layout.layout_five-up-left {
```

Comment:
@rpama question: other layout molecules have the `.flexbox` parent class specified here. Adding that class allows the layout max-width to work because of specificity (this effect probably wasn't intentional, but that's how it's currently working). Any reason you didn't include that here? I added it back in but wanted to check.